README.md

```diff
@@ -636,3 +636,37 @@ Compared to gomod and npm, Cachito support for pip has restrictions and limitati
 not expect. For more details, see the [Cachito pip documentation](docs/pip.md).
 
 [nexus-docs]: https://help.sonatype.com/repomanager3
+
+### git-submodule
+
+Cachito will produce a bundle that is downloadable at `/api/v1/requests/<id>/download`. This
```

Comment:
Could we have a sort of introductory first paragraph like we do for the other package managers? It could explain what git-submodule does and mention the fact that submodules are fetched *before* any of the other package managers are processed.

Or you could just reword this paragraph slightly, it currently does not feel like a "1st paragraph"

Reply:
Okay I agree.I had a few lines as an introductory paragraph in my draft but felt like it is only repeating stuff but I"ll try to do that.